lib/main_common.pm

```diff
@@ -1600,47 +1601,46 @@ sub load_filesystem_tests {
     loadtest 'console/snapper_used_space' if (is_sle('15-SP1+') || (is_opensuse && !is_leap('<15.1')));
 }
 
-sub load_wicked_tests {
-    loadtest 'wicked/before_test';
-    if (check_var('WICKED', 'basic')) {
-        loadtest 'wicked/basic/t01_basic';
-        loadtest 'wicked/basic/t02_static_addresses_legacy';
-        loadtest 'wicked/basic/t03_static_addresses_xml';
-        loadtest 'wicked/basic/t04_dynamic_addresses_legacy';
-        loadtest 'wicked/basic/t05_dynamic_addresses_xml';
-        loadtest 'wicked/basic/t06_static_routes_legacy';
-        loadtest 'wicked/basic/t07_static_routes_xml';
-    }
-    elsif (check_var('WICKED', 'advanced')) {
-        my $f = get_var('IS_WICKED_REF') ? 'ref' : 'sut';
-        loadtest 'wicked/advanced/' . $f . '/t01_gre_tunnel_legacy';
-        loadtest 'wicked/advanced/' . $f . '/t02_gre_tunnel_xml';
-        loadtest 'wicked/advanced/' . $f . '/t03_sit_tunnel_legacy';
-        loadtest 'wicked/advanced/' . $f . '/t04_sit_tunnel_xml';
-        loadtest 'wicked/advanced/' . $f . '/t05_ipip_tunnel_legacy';
-        loadtest 'wicked/advanced/' . $f . '/t06_ipip_tunnel_xml';
-        loadtest 'wicked/advanced/' . $f . '/t07_tun_interface_legacy';
-        loadtest 'wicked/advanced/' . $f . '/t08_tun_interface_xml';
-        loadtest 'wicked/advanced/' . $f . '/t09_tap_interface_legacy';
-        loadtest 'wicked/advanced/' . $f . '/t10_tap_interface_xml';
-        loadtest 'wicked/advanced/' . $f . '/t11_bridge_interface_legacy';
-        loadtest 'wicked/advanced/' . $f . '/t12_bridge_interface_xml';
-    }
-    elsif (check_var('WICKED', 'startandstop')) {
-        my $f = get_var('IS_WICKED_REF') ? 'ref' : 'sut';
-        loadtest 'wicked/startandstop/' . $f . '/t01_standalone_card_ifdown_ifreload';
-        loadtest 'wicked/startandstop/' . $f . '/t02_bridge_ifreload';
-        loadtest 'wicked/startandstop/' . $f . '/t03_bridge_ifup_ifreload';
-        loadtest 'wicked/startandstop/' . $f . '/t04_bridge_ifup_remove_all_config_ifreload';
-        loadtest 'wicked/startandstop/' . $f . '/t05_bridge_ifup_remove_one_config_ifreload';
-        loadtest 'wicked/startandstop/' . $f . '/t06_bridge_ifdown_create_new_config_ifreload_ifdown_ifup';
-        loadtest 'wicked/startandstop/' . $f . '/t07_bridge_ifdown_remove_one_config_ifreload_ifdown_ifup';
-        loadtest 'wicked/startandstop/' . $f . '/t08_sit_tunnel_ifdown';
-        loadtest 'wicked/startandstop/' . $f . '/t09_openvpn_tunnel_ifdown';
-        loadtest 'wicked/startandstop/' . $f . '/t10_vlan_ifup_all_ifdown_one_card';
+sub get_wicked_tests {
+    my (%args)     = @_;
+    my $basedir    = $bmwqemu::vars{CASEDIR} . '/tests/';
+    my $wicked_dir = $basedir . 'wicked/';
+    my $suite = get_var('WICKED') || die 'Missing WICKED variable';
+    my $type = get_var('IS_WICKED_REF') ? 'ref' : 'sut';
+    my $exclude   = get_var('WICKED_EXCLUDE', '$a');
```

Comment:
why default is '$a' ?

Reply:
`$a` is a regex which will never match. So this say we wonne launch all tests.